website/static/js/filepage/index.js

```diff
@@ -198,12 +199,17 @@ var FileViewPage = {
         $('.file-view-panels').removeClass().addClass('file-view-panels').addClass(fileViewPanelsLayout);
 
         m.render(document.getElementById('toggleBar'), m('.btn-toolbar[style=margin-top:20px]', [
-            ctrl.canEdit() ? m('.btn-group', {style: 'margin-left: 0;'}, [
-                m('.btn.btn-sm.btn-danger.file-delete', {onclick: $(document).trigger.bind($(document), 'fileviewpage:delete')}, 'Delete')
+            ctrl.viewUrl ? m('.btn-group', {}, [
+                m('a', {href: ctrl.viewUrl, target: '_blank', class: 'btn btn-sm btn-success'}, 'View ', [
```

Comment:
The other suggestion I have is to change the blue "View" text to "Content", but this also is less than ideal.